playwright/tests/request-blocking.spec.ts

```diff
@@ -0,0 +1,16 @@
+import * as selectors from '../data/selectors.json';
+import { expect, test } from '@playwright/test';
+
+test.describe('Request Blocking', () => {
+
+    test.beforeEach(async ({page}) => {
+        await page.goto('');
+
+        await page.route("**/users?page=2", route => route.abort());
+        await page.click(selectors.users);
+    });
+
+    test("should not let the response text appear in the UI's response section", async ({page}) => {
```

Comment:
`await expect(page.locator(selectors.uiUsersResponse)).toBeHidden();` This can be true without any action on the API level. I assume this should not be hidden before playing with our request.

I would suggest restructuring the test case to validate the entry state as well.
Something like:
1. UI verification
2. Fire the request and abort it
3. UI verification